leetcode/generate_md.py

```diff
@@ -1,3 +1,5 @@
 print('hello world!')
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your commit title looks better. Now it is: 
 Added code for reading file 
 Mistakes: 
 Where have you found such homework(Added)? It is not from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS'].
There is only one group(code), how could you make mistake here? So, it is definetly not 1013.
Wait...What did you do(for)? It is not from ['Added', 'Deleted'].